r/src/array__to_vector.cpp

```diff
@@ -434,6 +434,8 @@ class Converter_Time : public Converter {
       case TimeUnit::NANO:
         return 1000000000;
     }
+
+    return 0;
```

Comment:
```suggestion
    /* never happens */
    return 0;
```

Or should it return after the switch to avoid having to have this comment ?

Reply:
Moved to switch.